### ISSUE TITLE
ImportError: cannot import name Client

### ISSUE THREAD

**Issue by farid-project** [ping/instagram_private_api#248]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- Rlease read them **carefully** and answer completely.
- Do not post screenshots of error messages or code.
- Put an `x` into all the boxes [ ] relevant to your issue (==> [x] *no* spaces).
- Use the *Preview* tab to see how your issue will actually look like.
- Issues about reverse engineering is out of scope and will be closed without response.
- Any mention of spam-like actions or spam-related tools/libs/etc is strictly **not allowed**.

---

### Before submitting an issue, make sure you have:
- [x] Updated to the lastest version v1.6.0
- [x] Read the [README](https://github.com/ping/instagram_private_api/blob/master/README.md) and [docs](https://instagram-private-api.readthedocs.io/en/latest/)
- [x] [Searched](https://github.com/ping/instagram_private_api/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] Reviewed the sample code in [tests](https://github.com/ping/instagram_private_api/tree/master/tests) and [examples](https://github.com/ping/instagram_private_api/tree/master/examples)

### Which client are you using?
- [ ] app (``instagram_private_api/``)
- [x] web (``instagram_web_api/``)

---

### Describe your Question/Issue:

When I'm trying to use the web api python throw me an error "ImportError: cannot import name Client"

---

Paste the output of ``python -V`` here: Python 2.7.17

Code:

```python
import instagram_web_api
"highlight:18031606450087811"

user_name = 'xxxxxxxxxxx'
password = 'xxxxxxxxxxx'

# Some endpoints, e.g. user_following are available only after authentication
authed_web_api = instagram_web_api.Client(
    auto_patch=True, authenticate=True,
    username=user_name, password=password)

following = authed_web_api.highlight_reels('2719560079')
print following
```

Error/Debug Log:

```python
Traceback (most recent call last):
  File "instagram_web_api.py", line 1, in <module>
    import instagram_web_api
  File "/Users/jorgefaridpalenciafernandez/projects/testing/instagram_private_api/instagram_web_api.py", line 7, in <module>
    from instagram_web_api import Client, ClientCompatPatch, ClientError, ClientLoginError
ImportError: cannot import name Client
```

**Comment by wlerin** [comment]
Did you install it properly? Either by following the installation instructions or running 
`pip install -e .` in the cloned repo?

**Comment by farid-project** [comment]
> Did you install it properly? Either by following the installation instructions or running
> `pip install -e .` in the cloned repo?

It works great with instagram_private_api so yes, I think I've installed it properly.

**Comment by wlerin** [comment]
That doesn't answer my question.

`.../projects/testing/instagram_private_api/instagram_web_api.py`

Are you trying to run this inside the cloned repo? Again, install it properly.

**Comment by farid-project** [comment]
> That doesn't answer my question.
> 
> `.../projects/testing/instagram_private_api/instagram_web_api.py`
> 
> Are you trying to run this inside the cloned repo? Again, install it properly.

Ok, that was the problem, changing de filename solved the problem. Thanks so muchas for the help!!!
